cascade/src/main/java/me/saket/cascade/CascadePopupMenu.kt

```diff
@@ -85,7 +83,11 @@ open class CascadePopupMenu @JvmOverloads constructor(
     }
 
     showMenu(menuBuilder, goingForward = true)
-    popup.showAsDropDown(anchor, 0, 0, gravity)
+    if(atLocation) {
+      popup.showAtLocation(anchor, gravity, anchor.x.toInt(), anchor.y.toInt() + anchor.height)
```

Comment:
```suggestion
      val outLocation = IntArray(2)
      anchor.getLocationOnScreen(outLocation)
      popup.showAtLocation(anchor, gravity, outLocation[0], outLocation[1] + anchor.height)
```

Reply:
You should get View's absolute position here.

Reply:
Yup, that makes sense.
Especially for items in a recyclerview.
Did the changes.